src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -674,6 +684,91 @@ def move_mergeable_units(merge_directory, corpus_directory):
     shell.move(unit_path, dest_path)
 
 
+def pick_strategies(strategy_pool,
+                    fuzzer_path,
+                    corpus_directory,
+                    existing_arguments,
+                    minijail_chroot=None):
+  """Pick strategies."""
+  # Strategy pool is the list of strategies that we attempt to enable, whereas
```

Comment:
This comment needs to go where strategy_pool was defined.

Reply:
Done.